components/builder-worker/src/runner/studio.rs

```diff
@@ -110,7 +111,9 @@ impl<'a> Studio<'a> {
             }
         }
         cmd.stdout(Stdio::piped());
-        cmd.stderr(Stdio::piped());
+        // TED TODO: This will not work on windows. A more robust threading solution will be required for log_pipe
+        // to support consuming stderr and stdout
```

Comment:
Minor - I'd also add a comment on the hang behavior we encounter if we process them in sequence

Reply:
I really wish I had a more technical understanding of what happened but I can definitely describe the behavior